geode-redis/README.md

```diff
@@ -159,54 +159,54 @@ start server \
-|                    	| SETRANGE                                           	| CLUSTER DELSLOTS     	            |
-|                    	| SHUTDOWN                                           	| CLUSTER FAILOVER              	|
-|                    	| SINTER                                             	| CLUSTER FLUSHSLOTS                |
-|                    	| SINTERSTORE                                        	| CLUSTER FORGET                    |
-|                    	| SISMEMBER                                          	| CLUSTER GETKEYSINSLOT             |
-|                    	| SLOWLOG                                            	| CLUSTER INFO                  	|
-|                    	| SMOVE                                              	| CLUSTER KEYSLOT                   |
-|                    	| SPOP                                               	| CLUSTER MEET                      |
-|                    	| SRANDMEMBER                                        	| CLUSTER MYID                   	|
-|                    	| SSCAN                                              	| CLUSTER NODES                 	|
-|                    	| STRLEN                                             	| CLUSTER REPLICAS                  |
-|                    	| SUNION                                             	| CLUSTER REPLICATE              	|
-|                    	| SUNIONSTORE                                        	| CLUSTER RESET                  	|
-|                    	| TIME                                               	| CLUSTER SAVECONFIG                |
-|                    	| UNLINK [1]                                         	| CLUSTER SET-CONFIG-EPOCH          |
+| DECR               	| BITPOS                                             	| ACL GENPASS                   	|
+| DEL                	| DBSIZE                                             	| ACL GETUSER                   	|
+| EXISTS             	| DECRBY                                                | ACL HELP                      	|
+| EXPIRE             	| ECHO                                               	| ACL LIST                      	|
+| EXPIREAT           	| FLUSHALL                                           	| ACL LOAD                      	|
+| GET                 	| FLUSHDB                                            	| ACL LOG                       	|
+| GETRANGE              | GETBIT                                             	| ACL SAVE                      	|
+| HDEL        	     	| GETSET                                                | ACL SETUSER                   	|
+| HEXISTS           	| HINCRBY                                            	| ACL USERS                     	|
+| HKEYS                	| HSCAN                                                 | ACL WHOAMI                    	|
+| HGET               	| INCR                                                  | BGREWRITEAOF                  	|
```

Comment:
`INCR` should be moved from the Unsupported column to the Supported column.

Reply:
thanks-  changed